astropy/coordinates/tests/test_velocity_corrs.py

```diff
@@ -393,3 +396,36 @@ def _get_barycorr_bvcs_withvels(coos, loc, injupyter=False):
                            elevation=loc.geodetic[2].to(u.m).value)
         bvcs.append(res)
     return bvcs*u.m/u.s
+
+
+@pytest.mark.remote_data
+def test_regression_XXXX():
+    """
+    Make sure that when we include the proper motion and radial velocity of
+    a SkyCoord, our velocity corrections remain close to TEMPO2.
+
+    We check that tau Ceti is within 5mm/s
+    """
+    # Wright & Eastman (2014) Table2
+    # Corrections for tau Ceti
+    reduced_jds, tempo2, barycorr = np.loadtxt(
+        download_file('http://astroutils.astronomy.ohio-state.edu/exofast/pro/exofast/bary/zb.txt')).T
+
+    # tau Ceti Hipparchos data
+    tauCet = SkyCoord('01 44 05.1275 -15 56 22.4006',
+                      unit=(u.hour, u.deg),
+                      pm_ra_cosdec=-1721.05*u.mas/u.yr,
+                      pm_dec=854.16*u.mas/u.yr,
+                      distance=Distance(parallax=273.96*u.mas),
+                      radial_velocity=-16.597*u.km/u.s,
+                      obstime=Time(48348.5625, format='mjd'))
+    # CTIO location as used in Wright & Eastmann
+    xyz = u.Quantity([1814985.3, -5213916.8, -3187738.1], u.m)
+    obs = EarthLocation(*xyz)
+    times = Time(reduced_jds + 2400000, format='jd')
```

Comment:
Bit silly, but I'd write it as `Time(2400000, reduced_jds, format='jd')` - never hurts to show best practice also in one's test cases.

Reply:
Lovely - will do